common/pulp/common/error_codes.py

```diff
@@ -84,6 +84,11 @@
 PLP0034 = Error("PLP0034", _("The distributor %(distributor_id)s indicated a failed response when "
                              "publishing repository %(repository_id)s."),
                 ['distributor_id', 'repository_id'])
+PLP0035 = Error("PLP0035", _("The ContentUnit model class %(class_name)s failed to define the "
```

Comment:
Is there a way to make this less technical. I don't think a use would know what a ContentUnit model class is or familiarity with the unit_key_fields attribute. I don't have a specific suggestion for how to improve it.